tst/testinstall/coll.tst

```diff
@@ -154,6 +154,12 @@ gap> SetEnumeratorSorted(M0, []);
 gap> RepresentativeSmallest(M0);
 Error, <C> must be nonempty to have a representative
 
+# SortedList 
+gap> SortedList([2, 1, 3]);
+[ 1, 2, 3 ]
+gap> SortedList([2, 1, 3], {x, y} -> y < x);
+[ 3, 2, 1 ]
```

Comment:
Minor (completely optional) suggestion: Even better would be to also test that the input is not modified. E.g. by passing an immutable list to the function.